packages/lesswrong/server/recombee/client.ts

```diff
@@ -272,7 +275,21 @@ const recombeeRequestHelpers = {
     }
 
     return interleavedPosts;
-  }
+  },
+
+  openHybridRecombeePerfMetric<T extends RecRequest>(firstRequest: T, secondRequest: T) {
+    const firstRequestType = firstRequest.constructor.name;
+
+    const opName = (firstRequest instanceof requests.RecommendNextItems || secondRequest instanceof requests.RecommendNextItems)
+      ? `batch_${firstRequestType}`
+      : `batch_${firstRequestType}_${firstRequest.bodyParameters().scenario ?? 'unknown'}_${secondRequest.bodyParameters().scenario ?? 'unknown'}`;
+
+    return openPerfMetric({
+      op_type: 'recombee',
```

Comment:
Might make the name more specific in case we ever want to log things other than get recommendations